### PR TITLE
test: add new tests for help component

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

Needs to be added colour tests for help component.
Create a new `helpClassic.feature` to cover classic story
No need to adde another tests for Heading and i18n components due to lack of difference between default and classic stories.

### Current behaviour

lack of colour tests for help component

### Checklist

<del> - [ ] Release notes (Conventional Commits)  </del> 
<del> - [ ] Unit tests  </del>
- [x] Cypress automation tests

<del> - [ ] Storybook added or updated </del>
<del> - [ ] Theme support </del>
<del> - [ ] Typescript `d.ts` file added or updated </del>

### Testing instructions

run:
- `npm start`
- `npm run test-cypress` and run `help.feature` and `helpClassic.feature`

### CODE REVIEW

File: cypress/features/regression/helpClassic.feature [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Feature: Help component
```
Comment (KatarzynaQA) [inline cypress/features/regression/helpClassic.feature]:
```suggestion
Feature: Help component classic story
```

File: cypress/features/regression/helpClassic.feature [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  I want to change Help component properties
```
Comment (KatarzynaQA) [inline cypress/features/regression/helpClassic.feature]:
```suggestion
  I want to change Help component classic properties
```

File: cypress/features/regression/helpClassic.feature [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+  Background: Open Help component page
```
Comment (KatarzynaQA) [inline cypress/features/regression/helpClassic.feature]:
```suggestion
  Background: Open Help component page classic story
```

File: cypress/features/regression/help.feature [hunk 4]
```diff
@@ -56,4 +56,9 @@ Feature: Help component
       | áéíóú¿¡üñ               |
       | !@#$%^*()_+-=~[];:.,?{} |
       | ÄÖÜßäöüß                |
-      | <>                      |
\ No newline at end of file
+      | <>                      |
+
+  @positive
+  Scenario: Verify default color for help icon
+    # When I open "Help" component page
```
Comment (KatarzynaQA) [inline cypress/features/regression/help.feature]: Add comment why it's commented

File: cypress/features/regression/helpClassic.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/regression/helpClassic.feature]: updated

File: cypress/features/regression/helpClassic.feature (hunk 2 above)
Comment (DlgSHi) [reply cypress/features/regression/helpClassic.feature]: updated

File: cypress/features/regression/help.feature (hunk 4 above)
Comment (DlgSHi) [reply cypress/features/regression/help.feature]: updated

File: cypress/features/regression/helpClassic.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/features/regression/helpClassic.feature]: updated

**Pull request merged**